src/main/java/List.java

```diff
@@ -71,46 +82,47 @@ public void listAdd(String sentence) {
         printSuccessfulAddMessage(numItems, taskList[numItems - 1]);
     }
 
-    public void markTask(String index) {
+
+    // check if index represented by string is valid, if valid, return index in integer form.
+    // else return -1.
+    private int parseIndex(String index) {
         int i;
```

Comment:
The use of the word I in this context is confusing as the readers would not know what i is representing